Livecheckables/transmission-cli.rb

```diff
@@ -0,0 +1,4 @@
+class TransmissionCli
+  livecheck :url   => "https://github.com/transmission/transmission-releases/",
+            :regex => /href=.+transmission-v?(\d+(?:\.\d+)+)\.t/i
```

Comment:
```suggestion
            :regex => /href=.+?transmission-v?(\d+(?:\.\d+)+)\.t/i
```

We want to make sure to use `href=.+?` (rather than the greedy `href=.+`), to avoid escaping the bounds of the `href` attribute.

Reply:
Sorry about that :(